Automattic/pocket-casts-ios
# Champion: display a bottom sheet

<img src="https://github.com/user-attachments/assets/2e489ce8-1734-4c63-ac38-1dad9dd4f640" width="300">

## To test

1. Run the app on a real device
2. Go to Profile > Settings > Developer > Set to Lifetime
3. Go to Profile > tap Account
4. Tap "Pocket Casts Champion"
5. Tap "Rate Pocket Casts"
6. ✅ This should open the review screen
7. Go back to the app
8. Go to Profile > Settings > Developer > Set to Paid Plus
9. Go to Profile > tap Account
10. Tap "Monthly"
11. ✅ Nothing should happen

### Themes

You can either test themes one by one or by opening `ChampionView`, enabling preview and checking one by one.

## Checklist
